tests/test_dht_experts.py

```diff
@@ -143,22 +141,23 @@ def test_uid_patterns():
 
 @pytest.mark.forked
 @pytest.mark.asyncio
-async def test_negative_caching():
-    peers = []
-    for i in range(10):
-        neighbors_i = [f'{LOCALHOST}:{node.port}' for node in random.sample(peers, min(3, len(peers)))]
-        peers.append(hivemind.DHT(initial_peers=neighbors_i, cache_locally=False, start=True))
+async def test_negative_caching(n_peers=10):
+    dht_kwargs = {'cache_locally': False}
+
+    peers = [hivemind.DHT(start=True, **dht_kwargs)]
```

Comment:
Maybe pass it directly?

Reply:
There are 3 places where the DHT is created in this test. It is convenient to have one place where we can define/change common arguments to all of them.

Reply:
I see your point, though I'm not sure if these instances are intended to have any other common arguments. But let's keep it as is though